css/style.css

```diff
@@ -1,19 +1,465 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+    --color-main: #E16337;
+    --form-font-size: 21px;
+    --form-color: #666666;
+}
 
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+.btn
+{
+    text-decoration: none;
+    border-radius: 10px;
+    background-color: var(--color-main);
+    color: white;
+}
+
+.btn-bg {
+    font-size: 30px;
+    padding: 20px 50px;
+}
+
+/*header*/
+
+nav {
+    display: flex;
+    padding: 20px 150px;
+    background-color: #C4C4C41A;
+}
+
+.nav-section {
+    width: 100%;
+}
+
```

Comment:
Your nav section should have good contrast in line with the WCAG which means Web Content Accessibility Guidelines.

![image](https://user-images.githubusercontent.com/81495872/184165000-2d874a0d-782f-4102-a0df-4f586dc78e9c.png)